src/types.ts

```diff
@@ -1467,10 +1467,14 @@ export type MessageBase<
   UserType = UnknownType
 > = MessageType & {
   id: string;
+  pinned: boolean;
```

Comment:
@AnatolyRugalev will pinned always exist on a message object? So if a message is sent without specifying pinned: true will it default to false?

Reply:
Yes